db/migrate/20180904124427_create_events.rb

```diff
@@ -0,0 +1,11 @@
+class CreateEvents < ActiveRecord::Migration[5.2]
+  def change
+    create_table :events do |t|
+      t.string :title
+      t.string :location_name
```

Comment:
Is it a good idea to have a separated table for locations ? For example if in a near future you need to filter by location ?

Reply:
j'ai pas compris, oui je vais avoir un search par mot clef et si le user cherche un ville ilest sensé tombé sur les concerts dans cette ville

Reply:
Ah yes, du coup je pense qu'il te faut une table location avec les adresses des salles, mais bon il faut que tu aies ces informations